docs/notebooks/figanos_multiplots.ipynb

```diff
@@ -0,0 +1,213 @@
+{
```

Comment:
<p>for me, the suptitle is on top of the titles here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/figanos/pull/136/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>